include/dlaf/eigensolver/backtransformation.h

```diff
@@ -0,0 +1,44 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2021, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+#include <blas.hh>
+#include "dlaf/eigensolver/backtransformation/mc.h"
+#include "dlaf/matrix/matrix.h"
+#include "dlaf/types.h"
+#include "dlaf/util_matrix.h"
+
+namespace dlaf {
+namespace eigensolver {
+
+/// Eigenvalue back-transformation implementation on local memory.
+///
+/// It computes Q C, where Q = HH(1) HH(2) ... HH(m-mb) (HH(j) is the House-Holder transformation (I - v
+/// tau vH) defined by the j-th element of tau and the HH reflector stored in the j-th column of the matrix V.
+///
+/// @param mat_c contains the (mxn) matrix C (blocksize (mbxnb)), while on exit it contains Q C.
```

Comment:
minor comment: I don't remember how we used to write the shape of matrices in comments, but I find `mbxnb` not so nice. What about the `mb by nb` used in LAPACK doc? @rasolca

Reply:
Don't remember... but a quick search showed only a usage of `m by n`-like syntax in a miniapp.
Usage of `x` is more common:
e.g. https://github.com/eth-cscs/DLA-Future/blob/d9344e3390e11f2364752964a75611f2233e5a63/include/dlaf/matrix/tile.h#L85
Anyway I would add the spaces `(m x n)`.